src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const properties = sourceString.split(';');
+  const result = {};
```

Comment:
The variable name 'result' is too generic according to the checklist. Consider using a more descriptive name, such as 'styleObject' or 'cssProperties', which better reflects the content it's supposed to hold.